lms/djangoapps/support/static/support/jsx/success.jsx

```diff
@@ -0,0 +1,50 @@
+/* global gettext */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Success({homepageUrl, dashboardUrl, isLoggedIn}) {
+  return (<div className="contact-us-wrapper">
+    <div className="row">
+      <div className="col-sm-12">
+        <h2>{gettext('Contact Us')}</h2>
+      </div>
+    </div>
+
+    <div className="row">
+      <div className="col-sm-12">
+        <p>{gettext('Thank you for submitting a request! We will contact you within 24 hours.')}</p>
+      </div>
+    </div>
+
+    <div className="row">
+      <div className="col-sm-12">
+        {isLoggedIn &&
+        <a
```

Comment:
This seems to include some repeated code. Perhaps you could assign the text and the url before the jsx and use them here instead.